compiler/ast/ast.go

```diff
@@ -331,31 +320,13 @@ type Assignment struct {
 	RHS  Expr   `json:"rhs"`
 }
 
-//XXX TBD: chance to nano.Duration
-type Duration struct {
-	Seconds int `json:"seconds"`
-}
-
-type DurationNode struct {
-	Type    string `json:"type"`
-	Seconds int    `json:"seconds"`
-}
-
-func (d *Duration) MarshalJSON() ([]byte, error) {
-	if d.Seconds == 0 {
-		return json.Marshal(nil)
+func DurationToPrimitive(sec int) *Primitive {
```

Comment:
If this is sticking around, `NewPrimitiveFromSeconds` might be a better name.

Reply:
Unused... deleting